tests/NodeTest.php

```diff
@@ -5,7 +5,7 @@
 
 class NodeTest extends PHPUnit\Framework\TestCase
 {
-    public static function setUpBeforeClass(): void
+    public static function setUpBeforeClass() : void
```

Comment:
Since when do we have a blank before the colon of the return type? This is neither the upstream formatting style nor our formatting style for Lychee. If possible, I would like avoid this change in formatting, because it generates unnecessarily large diffs. (Here and below).

Reply:
Oups, I may have F up the merge, because I thought I kept our notation.

Reply:
fixed.